### DISCUSSION TITLE
Unlock the power of Semantic Kernel with AI Toolbox

### DISCUSSION THREAD

**Discussion by akordowski** [microsoft/semantic-kernel#9304] [Show and tell]
Hi everyone!

I would like to introduce a project I have been working on for some time. The project is called [**AI Toolbox**](https://github.com/akordowski/ai-toolbox). It is a .NET SDK which is built upon the Semantic Kernel. The idea for this project was to make access to AI driven applications for .NET developers as easy as possible. It is in an early stage of development, therefore, I would appreciate it if you would take a look, give it a try and share your opinion.

And if you like the project, I would appreciate it if you could help me 'spread the word' so the project can get some attention. Thanks in advance!

Here you can find the [project repo](https://github.com/akordowski/ai-toolbox) and [documentation](https://akordowski.github.io/ai-toolbox/).

If you have any questions or suggestions, feel free to reach out.

**Comment by sophialagerkranspandey** [comment]
Thanks so much for sharing! I'll connect with you to chat more about spreading the word about it.
